CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+##  v1.12.0 (05/07/2020)
+
+**Common**
+
+* Changed the DiskCache implementation to pass the provided `olp::cache::CacheSettings::enforce_immediate_flush` flag to `leveldb::WriteOptions::sync`.
+* Added a new CMake option `OLP_SDK_ENABLE_DEFAULT_CACHE`. The flag disables the LevelDB and Snappy dependencies and the default cache implementation.
+* Added a new LevelDB environment. The new environment overrides the random access file; by this, we disable mmap system calls, which saves the runtime memory on mobile platforms.
+* Changed the cache `Remove` and `RemoveKeysWithPrefix` API to consider protected keys, i.e., the keys and values are not removed when protected.
+* Fixed a crash during the compaction API call when there is no disk cache present.
+* Fixed the race condition in iOS networking.
+* Added a new `olp::cache::DefaultCache::StorageOpenResult::ProtectedCacheCorrupted` status. The status returned when the protected cache failed to open.
+* Fixed the issue when provided `olp::cache::OpenOptions` in `olp::cache::CacheSettings` are ignored by the mutable cache.
+* Change the log level for a few repetitive and unimportant log messages from `INFO` to `DEBUG`.
+* Break the cycling dependency in the Android network implementation (HttpClient.java).
+* Extend the `olp::client::DefaultRetryCondition` to take into account `IO_ERROR`, `OFFLINE_ERROR`, `TIMEOUT_ERROR`, `NETWORK_OVERLOAD_ERROR` errors, and treat them as retriable.
+* Changed the size estimation of the protected cache; instead of traversing the whole `leveldb`, we get the cache size on disk.
+* Fixed the cache opening sequence. When the mutable cache failed to open - the error is returned.
+* Added handling of the `NSURLErrorDataNotAllowed` on iOS as OFFLINE_ERROR.
+* Added handling of the `NSURLErrorNetworkConnectionLost` on iOS as IO_ERROR.
+
+**olp-cpp-sdk-authentication**
+
+* Fixed the usage of the same nonce for authentication retry requests. Now all auth requests generate a new unique nonce.
+* Added a new `SignInApple` API. It used to sign in to the HERE platform using Apple Id.
```

Comment:
Should we already mark it as GA? We did not even test it properly. I would say we should still keep it as **Work in progress**